### DISCUSSION TITLE
Use tire pressure if available

### DISCUSSION THREAD

**Discussion by rikroe** [bimmerconnected/bimmer_connected#340] [Feature request]
The iX3 supports target/actual tire pressure for each tire. Could be intersting for HA sensors.
https://github.com/bimmerconnected/bimmer_connected/discussions/327#discussioncomment-1639385

**Comment by robsonke** [comment]
Yep, would be interested, the car tells me the same but it's good as a reminder through HA. I guess it's added on these models because of the higher need for proper pressure with an EV.
